CMakeLists.txt

```diff
@@ -60,6 +61,16 @@ if(BUILD_TESTING)
     )
     target_link_libraries(test_find c_utilities)
   endif()
+
+  ament_add_gtest(test_concat
+    test/test_concat.cpp
+  )
+  if(TARGET test_concat)
+    target_include_directories(test_concat PUBLIC
+      ${c_utilities_INCLUDE_DIRS}
+    )
+    target_link_libraries(test_concat c_utilities)
+  endif()
```

Comment:
does this `if(TARGET ...)` needs to be duplicated for every test you add ? Or could we perform the same
```
target_include_directories(target_name PUBLIC
  ${c_utilities_INCLUDE_DIRS}
)
target_link_libraries(target_name c_utilities)
```
for each target?